File: surf-api-core/surf-api-core-server/src/main/java/dev/slne/surf/surfapi/core/server/listener/CoreListenerManager.java
package dev.slne.surf.surfapi.core.server.listener;

import dev.slne.surf.surfapi.core.server.listener.packet.CorePacketListenerManager;

public final class CoreListenerManager {

  public static final CoreListenerManager INSTANCE = new CoreListenerManager();

  private CoreListenerManager() {
  }

  public void registerListeners() {
    CorePacketListenerManager.INSTANCE.registerListeners();
  }

  public void unregisterListeners() {
    CorePacketListenerManager.INSTANCE.unregisterListeners();
  }
}

File: surf-api-core/surf-api-core-server/src/main/kotlin/dev/slne/surf/surfapi/core/server/listener/CoreListenerManager.kt
package dev.slne.surf.surfapi.core.server.listener

import dev.slne.surf.surfapi.core.server.listener.packet.CorePacketListenerManager

object CoreListenerManager {
    fun registerListeners() {
        CorePacketListenerManager.registerListeners()
    }

    fun unregisterListeners() {
        CorePacketListenerManager.unregisterListeners()
    }
}
